### ISSUE TITLE
Support SV-COMP ReachSafety-Combined with BMC

### ISSUE THREAD

**Issue by AdamZsofi** [ftsrg/gazer#85]
I'll need to take a look at exactly what the problem is here.
It could be a performance issue, but could be handled so that this issue is resolved.
It may also have something to do with inlining.

**Comment by sallaigy** [comment]
Indeed, it seems that we support this category alright, we just have performance issues (these programs are quite large).
